cli/src/utils/compareSchema.ts

```diff
@@ -53,43 +61,43 @@ export function compareSchemas(
     }
 
     // Compare column properties
-    for (const column of targetColumns) {
-      const sourceProps = source.tables?.[table]?.columns?.[column] ?? {};
-      const targetProps = target.tables?.[table]?.columns?.[column] ?? {};
-
-      if (sourceProps.type !== targetProps.type) {
-        edits.push({
-          alter_column: {
-            table,
-            column,
-            type: targetProps.type,
-            references:
-              targetProps?.type === 'link' && targetProps?.name
-                ? generateLinkReference({
-                    column: targetProps.name,
-                    table: tableNameFromLinkComment(targetProps?.comment ?? '') ?? ''
-                  })
-                : undefined
-          }
-        });
-      }
+    // for (const column of targetColumns) {
```

Comment:
I think we can maybe remove this bit. It's causing issues and we don't make use of it AFAICT